Data/fe.yaml

```diff
@@ -5,7 +5,7 @@ meta:
   comment: maybe there is something to say here
   last_updated: 2020-04-28
   references:
-    @article{curtiss2013unicorn,
+    '@article{curtiss2013unicorn,
```

Comment:
Are you sure, that one needs the quotation marks around the references?
I would prefer it without them, but I do not know if we have actually tested the `yaml` parser against that.